### DISCUSSION TITLE
Direct paths

### DISCUSSION THREAD

**Discussion by moserroger** [croneter/PlexKodiConnect#1457] [General]
I acivated direct path. In the tutorial it says the followig:
Under Advanced, reset the Kodi databases with the red Partial or full reset of Database and PKC

How to do this?

**Comment by croneter** [comment]
Pkc settings -> Advanced -> Reset the Kodi database (in red) -> YES -> NO

**Reply by moserroger** [reply]
Thanks again. I was looking in the Kodi settings and didn't realize I needed to move down in the PKC settings as the entry was hidden.

Now it works! Thanks a lot for your patience!
